inc/cleanHead.php

```diff
@@ -62,8 +78,9 @@
  * cleanScriptTag
  *
  * Clean up output of <script> tags
+ * TODO: fix or remove (broken on wp > 5.0.0)
  */
-add_filter('script_loader_tag', function ($input) {
-    $input = str_replace("type='text/javascript' ", '', $input);
-    return str_replace("'", '"', $input);
-});
+// add_filter('script_loader_tag', function ($input) {
+//     $input = str_replace("type='text/javascript' ", '', $input);
+//     return str_replace("'", '"', $input);
+// });
```

Comment:
how important is this issue for us. it seems that wordpress does not generate valid html5 markup. do we want to fix this now, or leave for later?

Reply:
I would definitely prefer it without to have valid HTML5 markup! Just expected it would need more work and time to get it safely fixed and didn't want the comments floating around for long.